mtools/util/profile_collection.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, hostname='localhost', port=27017, database='test',
 
         self.coll_handle = mc[database][collection]
 
-        if self.coll_handle.count() == 0:
+        if self.coll_handle.count_documents() == 0:
```

Comment:
Since we don't need the full count and only want to know if >=1 documents exists, passing `limit=1` here would be more efficient.